images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -257,4 +257,9 @@ function Get-DacFxVersion {
     cd "C:\Program Files\Microsoft SQL Server\150\DAC\bin\"
     $dacfxversion = (./sqlpackage.exe /version)
     return "DacFx $dacfxversion"
-}
\ No newline at end of file
+}
+
+
+function Get-SwigVersion {
+    return "Swig $(swig -version)"
```

Comment:
It returns the whole output, not just the version
![image](https://user-images.githubusercontent.com/48208649/120517691-65501a00-c3d9-11eb-9e3f-8182ff60d5a4.png)

Reply:
I'm looking for ways to display only version information

Reply:
fixed